src/validators/validateCreateNotificationParams.ts

```diff
@@ -6,6 +6,11 @@ export const validateCreateNotificationParams = (
 ): Joi.ValidationResult => Joi.object().keys({
   notificationType: Joi.string().required(),
   toUserId: Joi.number().integer().min(1).required(),
+  context: Joi.object().keys({
+  }).pattern(
```

Comment:
I don't understand what the call to [keys](https://joi.dev/api/?v=17.4.0#objectkeysschema) is doing here. Do we need it? It appears to work without that call; is there something I'm missing?

```suggestion
  context: Joi.object().pattern(
```

Reply:
ya sure.
I was having trouble when I was doing it at first so I assumed Keys were required. I will check if it works without it.